awscli/customizations/s3/utils.py

```diff
@@ -406,19 +406,39 @@ def uni_print(statement, out_file=None):
     out_file.flush()
 
 
-def bytes_print(statement):
+def bytes_print(statement, stdout=None):
```

Comment:
This function deals specifically with stdout. It really wouldn't work using a different file object due to the lack of a `buffer` property.